### PR TITLE
Improve readme; add contributing guide; add license

### PR DESCRIPTION
**Pull request by crookse**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Contributing to Rhum
+
+## Bug Reports
+
+A bug is a *demonstrable problem* that is caused by the code in the repository. Good bug reports are extremely helpful, so thanks!
+If you want to report a bug, click [here](https://github.com/drashland/deno-drash/issues/new?assignees=&labels=bug&template=bug_report.md&title=).
+
+## Feature Requests
+Feature requests are welcome. But take a moment to find out whether your idea fits with the scope and aims of the project. It's up to *you* to make a strong case to convince the project developer of the merits of this feature. Please provide as much detail and context as possible. If you want to request a feature, click [here](https://github.com/drashland/deno-drash/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=).
```
Comment (ebebbington) [inline .github/CONTRIBUTING.md:9]: Href link points to Drash ;)

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+## Pull Requests
+
+Please **ask first** before embarking on any significant pull request (e.g. implementing features, refactoring code), otherwise you risk spending a lot of time working on something that the project's developers might not want to merge into the project.
+
+1. [Fork](https://help.github.com/articles/fork-a-repo/) the project, clone your fork, and configure the remotes:
+    ```bash
+    # Clone your fork of the repo into the current directory
+    git clone https://github.com/<your-username>/deno-drash.git
```
Comment (ebebbington) [inline .github/CONTRIBUTING.md:18]: Deno Drash in mentioned here too

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    # Navigate to the newly cloned directory
+    cd deno-drash
```
Comment (ebebbington) [inline .github/CONTRIBUTING.md:20]: Deno Drash in mentioned here too

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    # Assign the original repo to a remote called "upstream"
+    git remote add upstream https://github.com/drashland/deno-drash.git
```
Comment (ebebbington) [inline .github/CONTRIBUTING.md:22]: Deno Drash in mentioned here too

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (crookse) [reply .github/CONTRIBUTING.md]: made the change

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (crookse) [reply .github/CONTRIBUTING.md]: made the change

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (crookse) [reply .github/CONTRIBUTING.md]: made the change

File: .github/CONTRIBUTING.md (hunk 4 above)
Comment (crookse) [reply .github/CONTRIBUTING.md]: made the change

**Pull request merged**